packages/react-redux/src/slices/cartoSlice.d.ts

```diff
@@ -18,6 +18,7 @@ type FilterBasic = {
   values: FilterValues;
   owner?: string;
   params?: Record<string, unknown>;
+  ignore?: boolean;
```

Comment:
this flag has very specific use case, maybe more verbose name would be better like `ignoreByNonOwners` or `onlyApplyToOwner` or `noCrossWidgetFiltering` ?

ignore seems too broad and misleading (it's still taken into account, but only by owner widget)